src/Engine/RadiumEngine.hpp

```diff
@@ -185,6 +185,106 @@ class RA_ENGINE_API RadiumEngine
      */
     void popFboAndViewport();
 
+    /// \name Time Management.
+    /// {
+
+    /**
+     * \brief Toggles Real-time (on) or Constant (off) time flow.
+     *
+     * In Real-time mode, time flows according to the actually elapsed time
+     * between consecutive frames, while in constant mode time flows according
+     * to a fixed time step.
+     */
+    void setRealTime( bool realTime );
+
+    /**
+     * \brief Indicates if Real-time time flow is on (Constant-time is off) .
+     * \see isConstantTime
+     */
+    bool isRealTime() const;
+
+    /**
+     * \brief Indicates if Constant-time time flow is on (Real-time is off) .
+     * \see isRealTime
+     */
+    bool isConstantTime() const;
+
+    /**
+     * \brief Sets the time delta between two frames for Constant-time time flow.
+     * \param forceConstantTime If true, enforces Constant-time time mode by calling
+     *                          setRealTime(false).
+     *                          If false, doesn't change the time mode.
+     * \return true if Constant-time flow is used.
+     * \see isConstantTime
+     */
+    bool setConstantTimeStep( Scalar dt, bool forceConstantTime = false );
+
+    /**
+     * \brief Toggles ForwardBackward time flow.
+     *
+     * When ForwardBackward mode is disabled (default), time will flow through
+     * the time window and restart over.
+     * When ForwardBackward mode is enabled, time will flow back and forth
+     * in the time window.
+     * \note If `end` time is negative, the ForwardBackward mode doesn't apply
+     *       and time just flows indefinitely.
+     */
+    void setForwardBackward( bool mode );
+
+    /**
+     * Toggles on/off time flow.
+     */
+    void play( bool isPlaying );
```

Comment:
If this method toggle a state, why does it take a parameter. If you keep the method as this, change the comment. Else, write 2 methods without parameters : `play` and `stop` without